### PR TITLE
register to LocationFetcher in GGAplication

### PR DESCRIPTION
**Pull request by teamagam1**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GimelGimel/pull/14" target="_blank">ReviewNinja

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java [hunk 1]
```diff
@@ -30,7 +40,42 @@ public void onCreate() {
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java:40]: Use PreferenceUtil to read/write from preferneces.
It also exposes getInt/Float that will ease the usage !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java [hunk 2]
```diff
@@ -30,7 +40,42 @@ public void onCreate() {
 
         int serviceFrequencyMs = getResources().getInteger(
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: meant to place the last comment here.

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java [hunk 3]
```diff
@@ -30,7 +40,42 @@ public void onCreate() {
...
+        int locationMinUpdatesMs = getResources().getInteger(
+                R.integer.location_min_update_frequency_ms);
+        float locationMinDistanceM = Float.parseFloat(getResources().getString(
+                R.string.location_threshold_update_distance_m));
+
         GGMessagePollingService.startMessagePollingPeriodically(this, serviceFrequencyMs);
+        mLocationFetcher = new LocationFetcher(this, (LocationManager) getSystemService(Context.LOCATION_SERVICE), new LocationFetcher.LocationFetcherListener() {
+            @Override
+            public void onProviderDisabled(@LocationFetcher.ProviderType String locationProvider) {
+
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: Lets Log.v it !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java [hunk 4]
```diff
@@ -30,7 +40,42 @@ public void onCreate() {
...
+            }
+
+            @Override
+            public void onProviderEnabled(@LocationFetcher.ProviderType String locationProvider) {
+
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: lets log.v it

Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: !fix

Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: !fix

File: app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java [hunk 5]
```diff
@@ -72,7 +65,7 @@
     private MessageBroadcastReceiver mTextMessageReceiver;
     private MessageBroadcastReceiver mLatLongMessageReceiver;
 
-    private LocationFetcher mLocationFetcher;
+//    private LocationFetcher mLocationFetcher;
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java]: Remove unused comment

File: app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java [hunk 6]
```diff
@@ -103,7 +103,7 @@ public void onNewMessage(Message msg) {
     }
 
     public void putMyLocationPin(LocationSample location) {
-        PointSymbol pointSymbol = new PointImageSymbol("mipmap/target-center-red.png", 36, 36);
+        PointSymbol pointSymbol = new PointImageSymbol(getString(R.string.viewer_self_icon_assets_path), 36, 36);
```
Comment (BarAgam) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/fragments/ViewerFragment.java:106]: cool

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java (hunk 1 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java:40]: I've used mPrefs which comes from GGAplication.getPrefs() !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java (hunk 3 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java (hunk 4 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java]: !fixed

File: app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java (hunk 5 above)
Comment (teamagam1) [inline app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java]: !completed

**Comment by teamagam1** [comment]
!ninjastar

**Pull request merged**